source/connection-method/index.html.md.erb

```diff
@@ -52,6 +55,17 @@ Only select this for requests when your users do not initiate an action and your
   </div>
 </details>
 
+### Choose the right technology stack
+
+Before building your application, make sure you select technologies that will allow
+you to send all header data required for your connection method. For example, some popular load balancer
+products do not pass on users’ public IPs or ports.
```

Comment:
Perhaps `solutions` or `technologies` rather then `products`.

Reply:
I wonder if we could turn this into a positive case: 
`For example, check if you selected load balancer can pass users' public IP addresses and ports.`

Reply:
Have removed the word product - weren't keen on `solutions` or `technologies`